src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/StartTask.groovy

```diff
@@ -53,7 +53,13 @@ class StartTask extends AbstractServerTask {
             ArrayList<Task> applicationBuildTasks = new ArrayList<Task>()
 
             if (server.dropins != null && !server.dropins.isEmpty()) {
-                applicationBuildTasks += server.dropins
+                server.dropins.each { Object dropinObj ->
+                    if (dropinObj instanceof Task) {
+                        applicationBuildTasks += dropinObj
+                    } else if (dropinObj instanceof File) {
+                        appsToVerify += getBaseName(dropinObj.name)
+                    }
```

Comment:
I want to talk through this logic a bit more.  If you have dropins tasks or Files, don't they both need to go in the appsToVerify list?